docs/self-hosted/maintenance.md

```diff
@@ -31,6 +31,9 @@ Go back to the dashboard on the Replicated Management Console and restart the ap
 
 If your certificate requires intermediate certificates to be recognized by your browser and/or Composer you can paste them into your certificate text file. Please make sure they are sorted from leaf (your certificate) via intermediate certificate to root certificate at the bottom. Otherwise Replicated will not recognize the certificate in a combined file.
 
+> In order to replace the certificate, you will first need to restore the ability to upload new TLS certificates as described in the 
```

Comment:
This has to go into the `/docs/self-hosted/kubernetes-maintenance` page instead, because this is about the KOTS plugin of our Kubernetes version. 

This page is about the old Replicated Native version (based on Docker) that we've deprecated now. It looks like the first two paragraphs above already explain how to replace the certificate for this version.

Reply:
What an obvious mistake 🤦‍♂️ Thanks

Reply:
I ran into the same things only last month, so.. perfectly understandable 😀